spec/features/user_signs_out_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+require 'support/authentication_helper'
+
+feature 'user signs out', %Q{
+  As an authenticated user
+  I want to sign out
+} do
+
+#   Acceptance Criteria:
+# - I must be logged in
+# - There is a link to "sign out"
+# - I am signed out once I click "sign out" link
+
+  scenario 'user signs out of session' do
+    user = FactoryGirl.create(:user)
+    sign_in(user)
+
+    expect(page).to have_content "Sign Out"
```

Comment:
Since you're about to click the link "Sign Out", I'd say that this line ^ is probably a bit superfluous - it's not actually adding anything to the functionality or accuracy of the test. If there isn't a "Sign Out", the test will already fail on line 20!